.circleci/config.yml

```diff
@@ -142,7 +143,7 @@ jobs:
           name: Pre-load integration test images to minikube
           command: |
             docker build -f model-engine/model_engine_server/inference/pytorch_or_tf.base.Dockerfile \
-            --build-arg BASE_IMAGE=pytorch/pytorch:1.7.1-cuda11.0-cudnn8-runtime \
+            --build-arg BASE_IMAGE=python:3.8-slim \
```

Comment:
interesting, does it mean we could use this for torch images in docker-compose.yml?

Reply:
hm I'm not sure actually